### PR TITLE
Added unit testing

### PR DESCRIPTION
**Pull request by Shalom-uwamahoro**
# Description

Please include a summary of the change and which issue is fixed. 
Please also include relevant motivation and context. 
List any dependencies that are required for this change.

Perhaps answering the following questions will help create a good description:
* What is this?
This pull request adds unit tests for the Django models used in our project. These tests ensure that the models are functioning correctly, covering methods, relationships, and field validations. The goal is to validate the behavior of the models and prevent future issues by detecting bugs early in development.

* Why this?
Unit testing is essential for maintaining the integrity of our Django models, particularly as we expand and modify the codebase. It helps ensure that the models behave as expected and makes it easier to identify and fix potential issues before they affect production.

* How are you doing this? 
I have created unit tests for each model using Django's built-in TestCase class. The tests check the functionality of the models, including relationships (ForeignKeys, OneToOne, etc.), custom model methods, and field validation. The tests are designed to run as part of our continuous integration (CI) process to ensure ongoing code quality.

Fixes # (issue) [Provide a link to the related Jira Task]
https://marthamutuaa-1712301322460.atlassian.net/jira/software/projects/DFYP/boards/2/backlog?epics=visible&issueParent=10171

## Type of change

Specify the kind of PR/MR

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A: Manually ran the tests using Django’s test runner (python manage.py test) to verify that all model unit tests pass without issues.
- [x] Test B: Ensured that the unit tests cover different edge cases, such as empty field values and invalid data.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Marthamwongeli — APPROVED** [review]

**Review by Yaar-Nuul — APPROVED** [review]

**Review by Bridget-Kathure — APPROVED** [review]

**Review by shallo-nairat — APPROVED** [review]

**Review by Minu-Ochieng — APPROVED** [review]

**Review by Yaar-Nuul — APPROVED** [review]

File: mamamind/mother/models.py [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# from django.db import models
+
+
+# class Mother(models.Model):
```
Comment (Ndichu-shee) [inline mamamind/mother/models.py:4]: why have you commented this

File: mamamind/nurse_admin/models.py [hunk 2]
```diff
@@ -2,29 +2,53 @@
 from django.contrib.auth.models import User
 from hospital.models import Hospital
 from django.conf import settings
+from django.core.validators import RegexValidator
 
+
+# class NurseAdmin(models.Model):
```
Comment (Ndichu-shee) [inline mamamind/nurse_admin/models.py:8]: why have you commented this

**Review by Ndichu-shee — APPROVED** [review]
